plugins/modules/swim_workflow_manager.py

```diff
@@ -1351,18 +1353,16 @@ def get_diff_distribution(self):
             return self
 
         if len(device_uuid_list) == 0:
-            self.status = "failed"
-            self.msg = "Image Distribution cannot proceed due to the absence of device(s)"
+            self.status = "success"
+            self.msg = "No matched device(s) for swim image distribution task."
```

Comment:
Can we rephrase?
self.msg = "The SWIM image distribution task could not proceed because no eligible devices were found."

Reply:
updated.